### PR TITLE
Feature/add remove

### PR DESCRIPTION
**Pull request by MahdiAghaali**
##In this branch I did the following activities and changes:

- Added functionality for:

  1. Adding a task
  2. removing a task
  3. editing a task
  4. selecting a task
  5. removing all completed tasks

### CODE REVIEW

File: src/ToDoList.js [hunk 1]
```diff
@@ -12,10 +12,68 @@ export default class ToDoList {
...
     this.container.innerHTML = '';
     for (let i = 0; i < this.tasks.length; i += 1) {
+      this.tasks[i].id = i + 1;
       const { description, isCompleted, id } = this.tasks[i];
       const task = new Task(id, description, isCompleted);
       task.showTask(this.container);
+
+      const lastInput = document.querySelectorAll(`input[id ="${i + 1}"]`);
+      lastInput[0].addEventListener('keyup', (e) => {
+        console.log(e.target.value);
```
Comment (m05e5) [inline src/ToDoList.js:22]: - [ ] [ optional ] You are required to follow JS best practice and keep your code clean so kindly remove the `console.logs` in your JS files

File: src/toDoList.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    this.container.innerHTML = '';
+    for (let i = 0; i < this.tasks.length; i += 1) {
+      this.tasks[i].id = i + 1;
+      const { description, isCompleted, id } = this.tasks[i];
+      const task = new Task(id, description, isCompleted);
+      task.showTask(this.container);
+
+      const lastInput = document.querySelectorAll(`input[id ="${i + 1}"]`);
+      lastInput[0].addEventListener('keyup', (e) => {
+        console.log(e.target.value);
```
Comment (m05e5) [inline src/toDoList.js:22]: - [ ] [ optional ] You are required to follow JS best practice and keep your code clean so kindly remove the `console.logs` in your JS files

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @MahdiAghaali    🔥 

Good job on implementing the requirements. You smashed it. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/toDoList.js (hunk 2 above)
Comment (MahdiAghaali) [reply src/toDoList.js:22]: Done, thanks for pointing it out!

**Pull request merged**